make/upgrade.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+#
+#Copyright 2021 VMware, Inc.
+#SPDX-License-Identifier: BSD-2-Clause
+#
+CURRENTPATH=`pwd`
+SOURCEDIR=$CURRENTPATH/../../
+
+if [ "$EUID" -ne 0 ]
```

Comment:
is_root() {
   if [ $(id -u) -ne 0 ]; then
      echo "You must be a root user"
      exit 1
   fi
}